### ISSUE TITLE
Suggestion: Run example 

### ISSUE THREAD

**Issue by danrobi11** [user234683/youtube-local#183]
Hi. 
What do you think about the idea to add a simple `.bash_aliases` example in the `README.md` Usage section? 
Im afraid some users might struggle figuring out how to run it. 

As for example. Something like this would be pretty straightforward:

```
alias ytlo="cd ~/.local/bin/youtube-local-2.8.5/ && python3 server.py | (sleep 1;flatpak run org.netsurf_browser.NetSurf http://localhost:8080/)"
```

Anyhow, Thanks for your project `youtube-local`.

**Comment by Foriki** [comment]
I think a better way of running it would be creating a python virtual environment and then running it from there so it would look like this

first launch
```
cd youtube-local
python -m venv env
source env/bin/activate
pip install -r requirements.txt
python server.py
```
then once you wanna reactivate it you'll simply have to just source and start the server
```
cd youtube-local
source env/bin/activate
python server.py
```
I **think** making a virtual environment is the universal method

**Comment by telekrex** [comment]
I agree; A line or two explaining server.py (it's explained in HACKING.md) and a few suggestions of quick and simple ways to run it would be nice to have in the main README.md. Advanced users may know what to do and have each their own ideas, but not everyone is advanced _hackor_
